ui.h

```diff
@@ -4,7 +4,12 @@
 #include "vec.h"
 #include "tags.h"
 
-#include <GL/gl.h>
+#ifdef __linux
```

Comment:
It begins ! The first platform specific code, this is exiciting. I wonder how tough it'll be to get it to build on windows again.